docker/docker-compose.example.yml

```diff
@@ -45,4 +45,12 @@ services:
       - "9000:9000"
     volumes: 
       - "../app:/var/www/app:cached"
+phpmyadmin:
+    image: phpmyadmin/phpmyadmin
+    ports:
+        - "99:80"
+    networks: 
+      sf_network: ~
+    links:
+        - db 
```

Comment:
Is this inlined properly compared to the file above?

Reply:
yes you are right :)